tensorflow/python/kernel_tests/bias_op_test.py

```diff
@@ -128,7 +129,16 @@ def _testGradient(self, np_input, bias, dtype, data_format, use_gpu):
           input_tensor, np_input.shape, output_tensor, np_input.shape)
       bias_jacob_t, bias_jacob_n = tf.test.compute_gradient(
           bias_tensor, bias.shape, output_tensor, np_input.shape)
-
+         
+      # Test gradient of BiasAddGrad
+      bias_add_grad = gen_nn_ops.bias_add_grad(out_backprop=output_tensor,
```

Comment:
Instead of calling the underlying function, maybe calling "tf.Gradient(loss, bias)". So even if the underlying implementation changes, the test might still be valid.